backtrader/feeds/ccxt.py

```diff
@@ -121,16 +122,22 @@ def _fetch_ohlcv(self, fromdate=None):
 
         while True:
             dlen = len(self._data)
+
             for ohlcv in sorted(self.store.fetch_ohlcv(self.symbol, timeframe=granularity,
-                                                       since=since, limit=limit)):
+                                                       since=since, limit=limit, params=self.p.fetch_ohlcv_params)):
+
                 if None in ohlcv:
                     continue
 
                 tstamp = ohlcv[0]
+
+                # Prevent from loading incomplete data
+                if tstamp > (time.time() * 1000):
+                    continue
+
```

Comment:
This looks like ccxt issue https://github.com/ccxt/ccxt/issues/2206. Should we ask ccxt guys to fix it?

Reply:
It looks like that issue was assigned. So perhaps you can put this on hold until the fix is released and tested. 

I will continue to use this fix in my fork of yours as I happen to need it until the ccxt issue is fixed. 

BTW - Do you see this issue with the exchanges you use? I notice the ticket only mentions it affects "some" exchanges. Currently, I am testing on bitmex but that is only because it has a test server with paper trading.

Reply:
OK, let's put it on hold then. I might consider merging it if ccxt issue not fixed for a long time. Splitting this commit would also help.

I didn't see this issue before. I have PR #6 about it, but it also mentions bitmex.